src/web_interface/templates/database/database_advanced_search.html

```diff
@@ -54,7 +54,7 @@ <h4>Example queries:</h4>
             </p>
             <p>
                 With regular expression:<br />
-                <span style="font-size: 0.9em; font-family: 'Deja Vu Sans Mono', monospace">{"processed_analysis.crypto_code.summary": {"$options": "si", "$regex": "sha"}}</span><br />
+                <span style="font-size: 0.9em; font-family: 'Deja Vu Sans Mono', monospace">{"processed_analysis.software_components.summary": {"$options": "si", "$regex": "Linux Kernel 2.6"}}</span><br />
```

Comment:
if dot is interpreted as wildcard this sample might not be the best. Maybe we should just remove the "s" option